### PR TITLE
SEO-176113-Image-Alt-Text-JS

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the image alt text.

Regards,
Beryl.

### CODE REVIEW

File: js/Menu/Data-binding.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data-binding
-description: data binding
+title: Data-binding in JavaScript Menu | Syncfusion
```
Comment (Yvone-Atieno) [inline js/Menu/Data-binding.md:3]: Check title

File: js/Toolbar/Data-binding.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data-binding
-description: data binding
+title: Data-binding in JavaScript Toolbar | Syncfusion
```
Comment (Yvone-Atieno) [inline js/Toolbar/Data-binding.md:3]: And Control word in all titles if not added

File: js/Grid/Paging.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](paging_images/paging_img2.png)
+![Pager with query string in JS Grid](paging_images/paging_img2.png)
```
Comment (Yvone-Atieno) [inline js/Grid/Paging.md:60]: Use JavaScript instead of JS in all places

File: js/DropDownTree/Checkbox.md [hunk 4]
```diff
@@ -434,4 +434,4 @@ function onCheckUncheckAll(args) {
     
 {% endhighlight %}
 
-![](Checkbox_images/Checkbox_img2.png)
\ No newline at end of file
+![checkNode/uncheckNode in JavaScript DropdownTree](Checkbox_images/Checkbox_img2.png)
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:437]: CheckNode and uncheckNode in JavaScript DropdownTree

File: js/DropDownTree/Checkbox.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
 title: CheckBox with DropDownTree widget for Syncfusion Essential JS
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:3]: Check title

File: js/DropDownTree/Checkbox.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
-description: Describes about Checkbox functionalities in DropDownTree widget for Syncfusion Essential JS
+description: Learn here about about Checkbox functionalities in Syncfusion Essential JavaScript DropDownTree Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:4]: Check description
image

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/Menu/Data-binding.md (hunk 1 above)
Comment (berylouma) [reply js/Menu/Data-binding.md]: Updated

File: js/Toolbar/Data-binding.md (hunk 2 above)
Comment (berylouma) [reply js/Toolbar/Data-binding.md]: Updated

File: js/Grid/Paging.md (hunk 3 above)
Comment (berylouma) [reply js/Grid/Paging.md]: Updated

File: js/DropDownTree/Checkbox.md (hunk 4 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md]: Done

File: js/DropDownTree/Checkbox.md (hunk 5 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md]: Done

File: js/DropDownTree/Checkbox.md (hunk 6 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md]: Done

File: js/DropDownTree/Checkbox.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: CheckBox with DropDownTree widget for Syncfusion Essential JS
-description: Describes about Checkbox functionalities in DropDownTree widget for Syncfusion Essential JS
+title: Checkbox with JavaScript DropDownTree Control | Syncfusion
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:3]: Checkbox in JavaScript DropDownTree Control | Syncfusion

File: js/DropDownTree/Checkbox.md [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
+description: Learn here about Checkbox functionalities in Syncfusion Essential JavaScript DropDownTree Control, its elements, and more.
 platform: js
 control: DropDownTree
 documentation: ug
 keywords: Checkbox, dropdown,textModes, Delimiter
 api: /api/js/ejDropDownTree
 ---
 
-# Checkbox
+# Checkbox with JavaScript DropDownTree widget
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:12]: Checkbox in. Use Checkbox in in any other place you used with

File: js/GroupButton/Overview.md [hunk 9]
```diff
@@ -1,18 +1,18 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview in JavaScript GroupButton Control | Syncfusion
```
Comment (Yvone-Atieno) [inline js/GroupButton/Overview.md:3]: Overview of JavaScript GroupButton Control | Syncfusion

File: js/GroupButton/Overview.md [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
+description: Learn here about overview support in Syncfusion Essential JavaScript GroupButton Control, its elements, and more.
 platform: js
 control: GroupButton
 documentation: ug
 api: /api/js/ejgroupbutton
 ---
 
-# Overview
+# Overview in JavaScript GroupButton
```
Comment (Yvone-Atieno) [inline js/GroupButton/Overview.md:11]: Overview of

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/GroupButton/Overview.md (hunk 9 above)
Comment (berylouma) [reply js/GroupButton/Overview.md]: Done

File: js/GroupButton/Overview.md (hunk 10 above)
Comment (berylouma) [reply js/GroupButton/Overview.md]: Done

File: js/DropDownTree/Checkbox.md (hunk 8 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md]: Done

File: js/DropDownTree/Checkbox.md (hunk 7 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md]: Done

File: js/DropDownTree/Checkbox.md [hunk 11]
```diff
@@ -1,15 +1,15 @@
...
+description: Learn here about Checkbox functionalities in Syncfusion Essential JavaScript DropDownTree Control, its elements, and more.
 platform: js
 control: DropDownTree
 documentation: ug
 keywords: Checkbox, dropdown,textModes, Delimiter
 api: /api/js/ejDropDownTree
 ---
 
-# Checkbox
+# Checkbox in JavaScript DropDownTree widget
```
Comment (Yvone-Atieno) [inline js/DropDownTree/Checkbox.md:12]: Remove widget word

File: js/GroupButton/Overview.md [hunk 12]
```diff
@@ -1,18 +1,18 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview of JavaScript GroupButton Control | Syncfusion
+description: Learn here about overview support in Syncfusion Essential JavaScript GroupButton Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline js/GroupButton/Overview.md:4]: Learn here about overview of Syncfusion Essential JavaScript GroupButton Control, its elements, and more.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/DropDownTree/Checkbox.md (hunk 11 above)
Comment (berylouma) [reply js/DropDownTree/Checkbox.md:12]: Done

File: js/GroupButton/Overview.md (hunk 12 above)
Comment (berylouma) [reply js/GroupButton/Overview.md:4]: Done

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**